api/v1alpha1/mustgather_types.go

```diff
@@ -53,12 +53,13 @@ type MustGatherSpec struct {
 	// +kubebuilder:validation:Optional
 	ProxyConfig ProxySpec `json:"proxyConfig,omitempty"`
 
-	// A time limit for gather command to complete a floating point number with an optional suffix:
-	// "s" for seconds (the default), "m" for minutes, "h" for hours or "d" for days will default to: $DEFAULT_MUST_GATHER_TIMEOUT'
+	// A time limit for gather command to complete a floating point number with a suffix:
+	// "s" for seconds, "m" for minutes, "h" for hours, or "d" for days.
+	// Will default to 0s
```

Comment:
```suggestion
	// Will default to no time limit.
```
Open to getting this suggestion to be rejected, I had to lookup the behavior of the `timeout` command because I wasn't sure, but 0s == no timeout.